activity/dispatchers.go

```diff
@@ -0,0 +1,59 @@
+package activity
+
+import "github.com/awslabs/aws-sdk-go/gen/swf"
+
+//ActivityTaskDispatcher is used by the FSM machinery to
```

Comment:
hehe cut and paste there, will update.

On Wed, Apr 8, 2015 at 9:15 PM, Ryan Brainard <notifications@github.com>
wrote:

> In activity/dispatchers.go
> <https://github.com/sclasen/swfsm/pull/23#discussion_r28034356>:
>
> > @@ -0,0 +1,59 @@
> > +package activity
> > +
> > +import "github.com/awslabs/aws-sdk-go/gen/swf"
> > +
> > +//ActivityTaskDispatcher is used by the FSM machinery to
>
> used by the FSM machinery to...?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/sclasen/swfsm/pull/23/files#r28034356>.
>